.github/workflows/Parameters.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           print(params)
 
           data = {
-             '3.6': { 'icon': '🔴', 'until': '23.12.2021' },
+             '3.6': { 'icon': '🔴', 'until': '23.12.2021' },   # Black circle ⚫ for EOL versions.
```

Comment:
I propose to set Python 3.6 to a black icon in the next days for 1 month and then deprecate Python 3.6. Colors are then moved by one position.

Reply:
Agree. However, I think that should be an explicit PR.

Reply:
The change will be a separate PR. I just added the colored circle to have it in the file.